libs/resolver/npm/mod.rs

```diff
@@ -526,6 +526,18 @@ impl<
           {
             return Ok(resolved);
           }
+          // failed to find types, just fall back to normal resolution
```

Comment:
Although it won't happen, it might be worthwhile to add a debug assertion that the current NodeResolutionKind is Types in case the code changes somehow.